docs/manual/index-checker.tex

```diff
@@ -207,7 +207,7 @@ \section{Lower bounds\label{index-lowerbound}}
     Checker.  On the left is a type system for lower bounds.  On the right
     is a type system for upper bounds.  Qualifiers written in gray should
     never be written in source code; they are used internally by the type
-    system. ''myArray'' in the Upper Bound qualifiers is an example of
+    system. \<"myArray"> in the Upper Bound qualifiers is an example of
```

Comment:
Why were these quotes changed from what was already in the manual? Was there something wrong with what was here already? It seems unrelated to the rest of the PR.

Reply:
I don't know, this is from https://github.com/typetools/checker-framework/pull/1461/commits/ec20ff96bc689003a47c05e15449c20e9bdf5d95.

Reply:
Okay, we'll leave it.